check-tables.sql

```diff
@@ -1,3 +1,3 @@
 \c nc_news_test
 
-SELECT articles.article_id, articles.title, articles.topic, articles.author, articles.created_at, articles.votes, articles.article_img_url, COUNT(comments.article_id) AS comment_count FROM articles LEFT JOIN comments ON articles.article_id = comments.article_id WHERE articles.votes >= 0 GROUP BY articles.article_id;
\ No newline at end of file
+-- SELECT articles.article_id, articles.title, articles.topic, articles.author, articles.created_at, articles.votes, articles.article_img_url, COUNT(comments.article_id) AS comment_count FROM articles LEFT JOIN comments ON articles.article_id = comments.article_id WHERE articles.votes >= 0 GROUP BY articles.article_id;
```

Comment:
This file is just for experimenting, as such it shouldn't be part of the repo on github (locally is fine!) You can remove it from the repo, then add the file name to the .gitignore and make it again to have an untracked version of it